include/dali/c_api.h

```diff
@@ -57,14 +57,26 @@ typedef enum {
   DALI_BOOL     =  11,
 } dali_data_type_t;
 
+
+/**
+ * @brief DALI initialization
+ *
+ * Call this function to initialize DALI backend. It shall be called once per process.
```

Comment:
I would add a comment that consecutive calls are noops or it is just safe to call more than once.